ReactWindows/ReactNative.Shared/UIManager/IAccessible.cs

```diff
@@ -0,0 +1,20 @@
+namespace ReactNative.UIManager
+{
+    /// <summary>
+    /// Partial ReactXP accessibility support: accessibilityTraits and importantForAccessibility.
```

Comment:
>ReactXP [](start = 16, length = 7)

We shouldn't reference ReactXP in the react-native-windows source code because react-native-windows doesn't depend on ReactXP.

Reply:
thanks! removed all ReactXP mentions from my changes.